src/functionalTest/resources/scenarios/RWA-340-Allocate-a-case-to-caseworker-negative-scenario2-midevent.json

```diff
@@ -0,0 +1,31 @@
+{
+  "description": "RWA-340 Allocate a case to caseworker when there is no caseworkers' names midEvent. //fixme: RWA-337",
+  "disabled": "true",
+  "request": {
+    "uri": "/asylum/ccdMidEvent",
+    "credentials": "CaseOfficer",
+    "input": {
+      "eventId": "allocateTheCase",
+      "state": "appealSubmitted",
+      "securityClassification": "PUBLIC",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "allocateTheCaseTo": "caseworker",
+          "caseWorkerLocationList": "765324"
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": ["There are no caseworkers for the selected location. Select a different location."],
```

Comment:
According to the story description, the error text should be "There are no users for the location you have selected. Choose another location to continue."